docs/guides/azure_resource_concept.md

```diff
@@ -1,27 +1,32 @@
-# Build ResourceId
+---
+layout: "azapi"
+page_title: "AzAPI Provider: Understand Azure resource concepts"
+description: |-
+  This guide will cover some basic Azure Resource concepts
+---
+# Azure Resource Concepts
 
-## Definitions
+## Resource Terminology
 1. `Root Resource / Top-Level Resource`: A resource with only a single level of nested types (i.e. there's just a single type after the RP namespace). `Microsoft.Network/networkSecurityGroups` is a top-level resource, whereas `Microsoft.Network/networkSecurityGroups/securityRules` is not.
 2. `Child Resource / Nested Resource`: A resource with two or more levels of nested types.
 3. `Parent Resource`: The parent to a child resource, identified by removing a level of nesting from the resource type. `Microsoft.Network/networkSecurityGroups` is the parent to `Microsoft.Network/networkSecuriyGroups/securityRules`.
 
 ## Scopes:
-Scopes are defined in `bicep-types-az`, however, there're some edge cases like a resource can be deployed in both extension
-and ResourceGroup scope, or a resource's scope is unknown.
+Every resource ID begins with a root scope. There are a few different types of root scopes:
 1. Tenant Scope: id starts with `/`, then followed with `/providers/Microsoft.Foo/bar/{name}`
 2. Subscription Scope: id starts with `/subscriptions/{subscriptionId}`, then followed with `/providers/Microsoft.Foo/bar/{name}`
 3. ResourceGroup Scope: id starts with `/subscriptions/{subscriptionId}/resourceGroups/{groupName}`, then followed with `/providers/Microsoft.Foo/bar/{name}`
-4. Extension Scope: id starts with `{resourceId}`, then followed with `/providers/Microsoft.Foo/bar/{name}`
+4. Extension Scope: id starts with `{resourceID}`, then followed with `/providers/Microsoft.Foo/bar/{name}`
 5. ManagementGroup Scope: id starts with `/providers/Microsoft.Management/managementGroups/{managementGroupName}/`, then followed with `/providers/Microsoft.Foo/bar/{name}`
```

Comment:
IIRC, there are also cases that a scope is ended with a location, shall we also mention that here (though it is not mentioned in the ARM wiki)?